src/settings/VoucherNumber/SettingsVoucherNumber.js

```diff
@@ -33,14 +37,31 @@ class SettingsVoucherNumber extends Component {
     super(props);
 
     this.configManager = props.stripes.connect(ConfigManager);
+    this.callout = React.createRef();
   }
 
-  beforeSave = (data) => JSON.stringify(data);
+  beforeSave = ({ voucherNumberPrefix, allowVoucherNumberEdit }) => {
+    return JSON.stringify({ voucherNumberPrefix, allowVoucherNumberEdit });
+  };
+
+  onReset = () => {
+    const { mutator, stripes } = this.props;
+    let { sequenceNumber } = getFormValues('configForm')(stripes.store.getState()) || {};
```

Comment:
IMHO, it's better to handle `sequenceNumber` through local Resource, or at last as local state. I don't feel like it's a good idea to mix it artificially into the form